webpack.config.js

```diff
@@ -28,6 +28,12 @@ module.exports = {
 							hotReload: false // pending https://github.com/sveltejs/svelte/issues/2377
 						}
 					}
+				},
+				{
+					test: /\.(png|svg|jpg|gif)$/,
```

Comment:
what about webp?

Reply:
I'm not sure if webpack's `file-loader` supports that or not. I just copied this from their docs. I see a separate project for webp, which makes me suspect it might not be supported out of the box: https://www.npmjs.com/package/webp-loader